From 5bafcca63f8d065461c311f1ab768e7d0d290813 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Wed, 22 Oct 2014 22:10:56 -0400
Subject: [PATCH] Ignore gdk-pixbuf deprecations

We're not going to start updating the 2.24 branch to newer
gdk-pixbuf APIs, so ignore deprecations.
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 85f0454..9e5c4f7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -621,8 +621,6 @@ if test "x$GLIB_VERSION_MAJOR_MINOR" = "x$GLIB_REQUIRED_VERSION_MAJOR_MINOR"; th
   CFLAGS="-DG_DISABLE_DEPRECATED $CFLAGS"
 fi
 
-CFLAGS="-DGDK_PIXBUF_DISABLE_DEPRECATED $CFLAGS"
-
 
 dnl
 dnl Check for bind_textdomain_codeset, including -lintl if GLib brings it in.
-- 
2.2.0

